lib/radiator_web/controllers/feed_controller.ex

```diff
@@ -7,7 +7,7 @@ defmodule RadiatorWeb.FeedController do
   @items_per_page 50
 
   def show(conn, %{"podcast_id" => id, "page" => page}) do
-    podcast = Directory.get_podcast!(id)
+    podcast = Directory.get_podcast!(id) |> Radiator.Repo.preload([:contributors])
```

Comment:
We probably want a `Directory.get_all_contributiors(podcast)` as well that is the union of the contributors of that podcast and all audio.

In terms of repo preloading: I think we should move those calls to the context, to make the context self contained and outside users not rely on knowing about the repo.

Reply:
Yes, controllers should never call Repo directly. This is temporary. Still good to call it out as "temporary" solutions have that sneaky habit of staying that way for longer than planned ;)